.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+name: CI
+
+on:
+  push:
+    branches: [ * ]
+  pull_request:
+    branches: [ * ]
```

This is the default for `branches`, so this can just be:
```suggestion
on:
  push:
  pull_request:
```